packages/hash/frontend/src/theme/components/inputs/MuiButtonThemeOptions.ts

```diff
@@ -14,114 +17,231 @@ export const MuiButtonThemeOptions: Components<Theme>["MuiButton"] = {
+      let focusStyles: CSSObject = { outline: "none" };
 
-      // The :focus-visible:after CSS styling applied to the button
-      const focusVisibleAfterStyles: CSSObject = {
+      // The :focus:after CSS styling applied to the button
+      const focusAfterStyles: CSSObject = {
         content: `""`,
         position: "absolute",
         left: -buttonFocusBorderOffset,
         top: -buttonFocusBorderOffset,
         bottom: -buttonFocusBorderOffset,
         right: -buttonFocusBorderOffset,
         border: `${buttonFocusBorderWidth}px solid`,
-        borderRadius: 6 + buttonFocusBorderOffset,
+        borderRadius:
+          (size === "large"
+            ? largeButtonBorderRadius
+            : baseButtonBorderRadius) + buttonFocusBorderOffset,
         borderColor: theme.palette.blue["70"],
       };
 
-      if (variant === "primary") {
-        /** ===== PRIMARY button specific styling ===== */
+      /** ====================== VARIANTS specific styling ============================= */
 
-        Object.assign(baseStyles, {
+      if (variant === "primary") {
+        /** ===== PRIMARY variant specific styling ===== */
+        baseStyles = {
+          ...baseStyles,
           color: theme.palette.common.white,
           background: theme.palette.blue[70],
-          backgroundColor: theme.palette.blue[70],
           position: "relative",
-          overflow: "hidden",
           zIndex: 0,
-          transition: "opacity 0.5s ease-in-out",
-          ...(size === "small" && {
-            padding: theme.spacing("8px", "20px"),
-          }),
-          ...(size === "medium" && {
-            padding: theme.spacing("12px", "28px"),
-            minHeight: 51,
-          }),
-        });
-
-        Object.assign(beforeStyles, {
+          transition: theme.transitions.create("opacity"),
+        };
+
+        beforeStyles = {
+          ...beforeStyles,
           position: "absolute",
           top: 0,
           width: "100%",
           height: "100%",
           opacity: 0,
           background:
             "linear-gradient(82.89deg, #9E56FA 5.64%, #644CFF 78.19%, #0070F4 121.05%)",
+          boxShadow: theme.boxShadows.purpleShadowMd,
+          transition: theme.transitions.create("opacity"),
           zIndex: -1,
-        });
+        };
+
+        hoverStyles = {
+          ...hoverStyles,
+          background: theme.palette.blue[70],
+        };
 
-        Object.assign(hoverBeforeStyles, {
+        hoverBeforeStyles = {
+          ...hoverBeforeStyles,
           opacity: 1,
-        });
-      } else if (variant === "tertiary_quiet") {
-        /** ===== TERTIARY button specific styling ===== */
+        };
+      } else if (variant === "secondary") {
+        baseStyles = {
+          ...baseStyles,
+          border: `1px solid ${theme.palette.blue[70]}`,
+          color: theme.palette.blue[70],
+          background: theme.palette.common.white,
+        };
+
+        hoverStyles = {
```

Comment:
**Blocking nit pick:** the primary and secondary buttons have different background colours on `:focus` in the designs:
<img width="821" alt="image" src="https://user-images.githubusercontent.com/42802102/158232051-b9a2b4e4-244a-4387-8eac-a7473e73f163.png">

Reply:
https://github.com/hashintel/hash/pull/384#discussion_r826674180

Reply:
Addressed here => cf1381b